caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -181,6 +191,66 @@ func parseTLS(h Helper) ([]ConfigValue, error) {
 					cp.Curves = append(cp.Curves, h.Val())
 				}
 
+			case "clients":
```

Comment:
```suggestion
			case "client_auth":
```

Makes more sense I think.